suite-native/device-authorization/src/deviceAuthorizationSlice.ts

```diff
@@ -67,7 +53,6 @@ export const deviceAuthorizationSlice = createSlice({
                 }
             })
             .addCase(UI.CLOSE_UI_WINDOW, state => {
-                state.hasDeviceRequestedPin = false;
```

Comment:
Touching on what I wrote in my comment earlier, I'm afraid that since we implemented this without the need for button requests, native code will now lack the removal of button requests after success / error and the selector `selectDeviceRequestedPin ` might return true even though it shouldn't. 

Going into the future, I'd consider some kind of "resetting" of this array when reacting `UI.CLOSE_UI_WINDOW` so it's not possible to have a hanging button request, that wasn't deleted manually.